src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -345,6 +346,9 @@ void WorldSession::HandleMovementOpcodes(WorldPacket& recvData)
         plrMover->SetInWater(!plrMover->IsInWater() || plrMover->GetBaseMap()->IsUnderWater(movementInfo.pos.GetPositionX(), movementInfo.pos.GetPositionY(), movementInfo.pos.GetPositionZ()));
     }
 
+    if (plrMover)
+        sAnticheatMgr->StartHackDetection(plrMover, movementInfo, Opcodes(opcode));
```

Comment:
The Opcodes cast looks unnecessary as opcode is already `uint16` and `StartHackDetection` takes an `uint16`